ws-style.gemspec

```diff
@@ -29,9 +29,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'rubocop-vendor', '>= 0.6.0'
 
   s.add_development_dependency 'bundler'
-  s.add_development_dependency 'bundler-audit'
+  s.add_development_dependency 'bundler-audit', '~> 0.9.1'
```

Comment:
You can probably remove the version here. It will stay more up to date with non version spec, and I believe this change was just to fix the `thor` dependency issue between `bundler-audit` and `keepachangelog`.